pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -300,17 +298,18 @@ func (c *cephStatusChecker) getRookPodsOnNode(node string) ([]v1.Pod, error) {
 		"csi-cephfsplugin-provisioner",
 		"csi-cephfsplugin",
 		"rook-ceph-operator",
+		"rook-ceph-mon",
+		"rook-ceph-osd",
+		"rook-ceph-crashcollector",
+		"rook-ceph-mgr",
+		"rook-ceph-mds",
 	}
 	podsOnNode := []v1.Pod{}
 	pods, err := c.context.Clientset.CoreV1().Pods(clusterName.Namespace).List(context.TODO(), metav1.ListOptions{})
```

Comment:
This seems to find all pods in the rook namespace. Where is the node filter applied? I don't see it in this method.

Reply:
Added.